web/src/components/tasks/cannabis/CannabisPriorityStatusTask.test.tsx

```diff
@@ -490,10 +490,10 @@ describe("<CannabisPriorityStatusTask />", () => {
 
       renderPage(task);
 
-      const eligibilityPhrase = templateEval(Config.cannabisPriorityStatus.phraseWithThreePriorities, {
-        priorityStatusOne: Config.cannabisPriorityTypes.minorityWomenOrVeteran,
-        priorityStatusTwo: Config.cannabisPriorityTypes.impactZone,
-        priorityStatusThree: Config.cannabisPriorityTypes.socialEquity,
+      const eligibilityPhrase = templateEval(Config.cannabisPriorityStatus.phrase3, {
+        type1: Config.cannabisPriorityTypes.minorityOrWomen,
```

Comment:
when you delete this from the config, this will have to go back to being `minorityWomenOrVeteran` (and in the other tests as well)